deny.toml

```diff
@@ -17,6 +17,11 @@ ignore = [
     # Used by alloy
     # proc-macro-error is unmaintained
     "RUSTSEC-2024-0370",
+    # instant is unmaintained
+    "RUSTSEC-2024-0384",
```

Comment:
This could have probably been part of a different PR

Reply:
unfortunately it was triggered in this CI :(

Reply:
still could have been a separate PR to main then merge main into this. Example: https://github.com/matter-labs/foundry-zksync/pull/564 was done as a deny issue happening on another PR. Keeps PR single themed.